### PR TITLE
Switch icon theme to Fluent icons

### PR DESCRIPTION
**Pull request by LorenDB**

### CODE REVIEW

File: resources/icons/ui/arrow-down.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M16 3C16.5523 3 17 3.44772 17 4V25.5858L25.2929 17.2929C25.6834 16.9024 26.3166 16.9024 26.7071 17.2929C27.0976 17.6834 27.0976 18.3166 26.7071 18.7071L16.7071 28.7071C16.3166 29.0976 15.6834 29.0976 15.2929 28.7071L5.29289 18.7071C4.90237 18.3166 4.90237 17.6834 5.29289 17.2929C5.68342 16.9024 6.31658 16.9024 6.70711 17.2929L15 25.5858V4C15 3.44772 15.4477 3 16 3Z" fill="#212121"/>
+</svg>
```
Comment (deepbluev7) [inline resources/icons/ui/arrow-down.svg:3]: This is used for downloads, right? Could you use the download icon instead?

File: resources/icons/ui/angle-arrow-down.svg [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<svg width="28" height="28" viewBox="0 0 28 28" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M4.21967 9.46967C4.51256 9.17678 4.98744 9.17678 5.28033 9.46967L14 18.1893L22.7197 9.46967C23.0126 9.17678 23.4874 9.17678 23.7803 9.46967C24.0732 9.76256 24.0732 10.2374 23.7803 10.5303L14.5303 19.7803C14.2374 20.0732 13.7626 20.0732 13.4697 19.7803L4.21967 10.5303C3.92678 10.2374 3.92678 9.76256 4.21967 9.46967Z" fill="#212121"/>
+</svg>
```
Comment (deepbluev7) [inline resources/icons/ui/angle-arrow-down.svg:3]: Do we use that anywhere?

File: resources/icons/ui/hamburger-menu.svg [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M16 9.5C14.6193 9.5 13.5 8.38071 13.5 7C13.5 5.61929 14.6193 4.5 16 4.5C17.3807 4.5 18.5 5.61929 18.5 7C18.5 8.38071 17.3807 9.5 16 9.5ZM16 18.5C14.6193 18.5 13.5 17.3807 13.5 16C13.5 14.6193 14.6193 13.5 16 13.5C17.3807 13.5 18.5 14.6193 18.5 16C18.5 17.3807 17.3807 18.5 16 18.5ZM13.5 25C13.5 26.3807 14.6193 27.5 16 27.5C17.3807 27.5 18.5 26.3807 18.5 25C18.5 23.6193 17.3807 22.5 16 22.5C14.6193 22.5 13.5 23.6193 13.5 25Z" fill="#212121"/>
+</svg>
```
Comment (deepbluev7) [inline resources/icons/ui/hamburger-menu.svg:3]: This should either keep the old name or be called "more" or "options"?

File: resources/icons/ui/image-loading.svg [hunk 4]
```diff
@@ -0,0 +1,3 @@
...
+</svg>
```
Comment (deepbluev7) [inline resources/icons/ui/image-loading.svg:3]: This is not really a loading indicator, more a failed indicator

File: resources/icons/ui/upload.svg [hunk 5]
```diff
@@ -0,0 +1,3 @@
...
+</svg>
```
Comment (deepbluev7) [inline resources/icons/ui/upload.svg:3]: Maybe this should be " Arrow Upload "?

Comment (deepbluev7) [reply resources/icons/ui/upload.svg:3]: Is this even used anywhere?

File: resources/icons/ui/arrow-down.svg (hunk 1 above)
Comment (LorenDB) [reply resources/icons/ui/arrow-down.svg:3]: Good point

**Pull request merged**